internal/dev_server/model/events.go

```diff
@@ -0,0 +1,14 @@
+package model
+
+// Event for individual flag overrides
+type UpsertOverrideEvent struct {
+	FlagKey    string
+	ProjectKey string
+	FlagState  FlagState
+}
+
+// Event for full project sync
+type SyncEvent struct {
+	ProjectKey    string
+	AllFlagsState FlagsState
```

Comment:
This got renamed from `FlagsState` to `AllFlagsState` just to add a smidge of clarity

Reply:
Nice! It was _very_ confusing.